lib/datasets.go

```diff
@@ -666,101 +599,33 @@ type datasetImpl struct{}
 
 // Get retrieves datasets and components for a given reference.t
 func (datasetImpl) Get(scope scope, p *GetParams) (*GetResult, error) {
-	res := &GetResult{}
-
-	scope.EnableWorkingDir(true)
-	ds, err := scope.Loader().LoadDataset(scope.Context(), p.Ref)
+	ref, ds, err := openAndLoadFSIEnabledDataset(scope, p)
```

Comment:
I ultimately decided it was probably overkill for this pr, but this is where I was curious if loading & opening the dataset should happen in a lower package